index.rst

```diff
@@ -1,52 +1,163 @@
 ..
-  Technote content.
-
   See https://developer.lsst.io/docs/rst_styleguide.html
   for a guide to reStructuredText writing.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+:tocdepth: 1
 
-  Use the following syntax for sections:
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Sections
-  ========
+.. sectnum::
 
-  and
+.. Add content below. Do not include the document title.
 
-  Subsections
-  -----------
+.. note::
 
-  and
+   **This technote is not yet published.**
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+   Summary of state of the art PSF estimation tools and their suitability for the LSST alert pipeline.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+Overview
+========
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
-     :target: http://target.link/url
+Understanding the point-spread function is important for several Alert Production tasks, including image differencing and photometry. The LSST point spread function is expected to vary in a complex way with chip position, telescope orientation and focus, and atmospheric conditions. The PSF must be fit independently for each image (in particular, the effect of atmospheric turbulence is expected to change even between the two snaps of a visit), and must be fit quickly enough to not form a bottleneck for Alert Production.
 
-     Caption text.
+The requirements for PSF quality depend on downstream operations, but are in general poorly quantified. In particular:
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+- the Alard & Lupton ([AL1998]_) image differencing algorithm requires that the FHWM of the PSF be known to within 10% (Reiss, priv. comm. 2017), but is otherwise self-contained
+- the ZOGY image differencing algorithm requires a "high quality" PSF (Reiss, priv. comm. 2017)
+- the limiting factor for our current DCR correction algorithm is noise in the images to be corrected, not errors in the PSF. Therefore, the only requirement DCR correction imposes is that the PSF not have color terms (Sullivan, priv. comm. 2017; [LDM151]_)
+- PSF requirements specific to LSST photometry are not available, but PSF photometry is in general only moderately sensitive to PSF models ([ABP2006]_), so photometry is unlikely to be the limiting application.
+- PSF requirements specific to LSST astrometry are not available
 
-   Feel free to delete this instructional comment.
+In addition, the Alert Pipeline PSF fitting code needs to be robust to crowded fields (up to source fill factor ~0.2; Bosch, April 2017 PST Meeting), and may need to support high-frequency atmospheric components ([CMJ2012]_), depending on the accuracy needed for image differencing.
```

Comment:
Difficult (since I don't know how he computed that number), and it's arguably a step in the wrong direction. The papers I'm looking at work on a variety of instruments with different pixel scales and FWHMs; talking about "fill factor" (however vaguely defined) makes it easier to discuss field crowding across different instruments than sources per square degree.